.github/workflows/validate.yml

```diff
@@ -67,15 +67,6 @@ jobs:
         with:
           branch: v${{ steps.version.outputs.value }}
 
-  # Fail on version collision. Every change to code requires an update to the version number.
```

Comment:
Removing this check because we should not enforce a 1:1 relationship between PRs and releases.